src/OSPSuite.Presentation/Presenters/Charts/CurveSettingsPresenter.cs

```diff
@@ -151,7 +157,9 @@ public void Refresh()
 
       public void NotifyCurvePropertyChange(CurveDTO curveDTO)
       {
-         CurvePropertyChanged(this, new CurveEventArgs(curveDTO.Curve));
+         this.DoWithinLatch(() =>
```

Comment:
Notify curve changed (eg. color changed or visible )) would trigger a rebind which is not required